<?xml version="1.0"?>
<launch>
  <arg name="robot_model" default="ur5e"/>

  <include file="$(find simple_ur5_controller)/launch/load_ur_pillar.launch" pass_all_args="true" />

  <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui"/>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find simple_ur5_controller)/cfg/view_robot.rviz" required="true"/>
</launch>
